src/coloris.js

```diff
@@ -154,7 +162,7 @@
       let reposition = { left: false, top: false };
       let offset = { x: 0, y: 0 };
       let left = coords.x;
-      let top =  scrollY + coords.y + coords.height + settings.margin;
+      let top = scrollY + coords.y + coords.height + settings.margin;
```

Comment:
Sorry, I already fixed this in a commit that I haven't yet pushed. Do you mind removing it?